tests/logprob/test_basic.py

```diff
@@ -58,16 +58,15 @@
 from pymc.logprob.basic import factorized_joint_logprob, icdf, joint_logp, logcdf, logp
 from pymc.logprob.utils import rvs_to_value_vars, walk_model
 from pymc.testing import assert_no_rvs
-from tests.logprob.utils import joint_logprob
 
 
 def test_joint_logprob_basic():
-    # A simple check for when `joint_logprob` is the same as `logprob`
+    # A simple check for when `factorized_joint_logprob` is the same as `logprob`
     a = pt.random.uniform(0.0, 1.0)
     a.name = "a"
     a_value_var = a.clone()
 
-    a_logp = joint_logprob({a: a_value_var}, sum=False)
+    a_logp = logp(a, a_value_var)
```

Comment:
Shouldn't this be `factorized_joint_logprob`?

Reply:
For single variables it's fine to use logp directly

Reply:
Ah it's suppose to be a test of factorized, then yes

Reply:
Yes but then doesn't the test compares two equal things?